integrationtests/server-integration/server-integration-commons/src/test/java/org/infinispan/test/integration/remote/AbstractHotRodQueryIT.java

```diff
@@ -86,26 +119,29 @@ public void testRemoteQuery() throws Exception {
     */
    @Test
    public void testUninverting() throws Exception {
-      rcm = createCacheManager();
-
-      SerializationContext serializationContext = MarshallerUtil.getSerializationContext(rcm);
-      ProtoSchemaBuilder protoSchemaBuilder = new ProtoSchemaBuilder();
-      String protoFile = protoSchemaBuilder.fileName("test.proto")
-            .addClass(Person.class)
-            .build(serializationContext);
-
-      RemoteCache<String, String> metadataCache = rcm.getCache(ProtobufMetadataManagerConstants.PROTOBUF_METADATA_CACHE_NAME);
-      metadataCache.put("test.proto", protoFile);
-      assertFalse(metadataCache.containsKey(ProtobufMetadataManagerConstants.ERRORS_KEY_SUFFIX));
-
-      RemoteCache<String, Person> cache = rcm.getCache();
-      cache.clear();
-
-      QueryFactory qf = Search.getQueryFactory(cache);
-      Query<Person> query = qf.from(Person.class)
-            .having("name").eq("John")
-            .orderBy("id")
-            .build();
-      Assert.assertEquals(0, query.execute().list().size());
+      ConfigurationBuilder config = new ConfigurationBuilder();
+      config.addServer().host("127.0.0.1");
```

Comment:
same as before. we can extract into a private method to avoid repeating and using constants